scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -527,20 +527,37 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm solve case</h5>
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">
-        Are you sure? Marking a case as Solved will queue it to delete analysis files
-        ahead of schedule, and disable the alignment view.
-        You will then have to request a FULL rerun to continue evaluating e.g. research variants.
-        <br><br>Marking Solved directly does not store the Causative variant.
-        Only use it for cases where the causation is not readily markable, such as UPD or SMN.
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-        <form method="POST"
+      <form method="POST"
           action="{{ url_for('cases.status', institute_id=institute._id, case_name=case.display_name) }}">
-          <button type="submit" class="btn btn-primary" value="solved" name="status">Solve</button>
-        </form>
+        <div class="modal-body">
```

Comment:
This works but I wonder why we put this modal in the collapsible actionbar in the first place..